docs/topics/mta-rn-known-issues-7-0-3.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: REFERENCE
+[id="rn-known-issues-7-0-3_{context}"]
+= Known issues
+
+{ProductFullName} version 7.0.3 has the following known issues.
+
+.IntelliJ IDE plugin Keymap actions do not work
+// Scheduled to be fixed in 7.0.3. Move this to resolved issues once fixed?
+
+In order to automate {ProductShortName}, the IntelliJ IDE plugin Keymap actions must work. They currently do not work.  link:https://issues.redhat.com/browse/MTA-2460[(MTA-2460)]
+
+.Eclipse IDE cannot run {ProductShortName} analysis
+
+// Get more information
+Eclipse IDE cannot run {ProductShortName} plugin and perform analysis. link:https://issues.redhat.com/browse/MTA-2563[(MTA-2563)]
+
+.Analysis fails on MacOS
+// Moved to resolved issues once fixed?
+
+Running the `analyze` command on a MacOS machine does not work when Podman does not have enough memory. The analysis failed with timeout, without returning an error. 
+
+Workaround: Set Podman memory to at least 4GB. Podman on MacOS sets default memory to 2GB, which is not enough to run {ProductShortName}. link:https://issues.redhat.com/browse/MTA-2518[(MTA-2518)]
+
+.Custom rules do not work
+// Need more information. I don't really understand what is happening.
+
+Running an analysis using a custom rule does not work. The link:https://issues.redhat.com/browse/MTA-2015[(MTA-2015)]
```

Comment:
If there's no workaround, I'd say so directly, for example "Currently, there is no workaround".

```suggestion
.Custom rules do not work during application analysis
// Need more information. I don't really understand what is happening.

Running an application analysis by using a custom rule does not work. Currently, there is no workaround. link:https://issues.redhat.com/browse/MTA-2015[(MTA-2015)]
```